run_inference.py

```diff
@@ -67,13 +67,16 @@ def main():
 
         output = disp_net(tensor_img)[0]
 
+        file_ = file.replace(args.dataset_dir, '')
```

Comment:
as file is a Path object we can use its sweet API to make it more readable see here (https://pathpy.readthedocs.io/en/stable/api.html). Also I'm not a huge fan of the `_` appendix here. 

```python
file_path, file_ext = file.relpath(args.dataset_dir).splitext()
file_name = '-'.join(file_path.splitall())
```